From 7938d91f62cfbbd3677b9950d2343d1fa6c036d8 Mon Sep 17 00:00:00 2001
From: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
Date: Thu, 31 Mar 2022 16:59:16 +0200
Subject: [PATCH 30/33] imx8mp-irma6r2.dts: Set DRAM size to 1x512MiB

For series production we use 512MiB DRAM. The 512MiB DRAM configuration also
works for 1GiB DRAMs.

Signed-off-by: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
---
 arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts b/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
index 4460e88180b1..28906e0ffc54 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
@@ -12,6 +12,11 @@
 	model = "IRMA6 Release2 i.MX8MPlus DDR4 board";
 	compatible = "fsl,imx8mp-evk", "fsl,imx8mp";
 
+	memory@40000000 {
+		device_type = "memory";
+		reg = <0x0 0x40000000 0 0x20000000>;
+	};
+
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
-- 
2.25.1

